tests/middleware/test_logging.py

```diff
@@ -154,3 +154,43 @@ async def app(scope, receive, send):
             if record.name == "uvicorn.access"
         ]
         assert '"GET / HTTP/1.1" 599' in messages.pop()
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "access_log_format,expected_output",
+    [
+        # TODO: add more tests
```

Comment:
I agree with the `# TODO: add more tests` comment. If we want to support all the formatting options independently of Gunicorn, we should test every one, and probably many combinations. See [Gunicorn `tests/test_logger.py`](https://github.com/benoitc/gunicorn/blob/933b210f2eff943f4cd9502c676f3d93c82f171c/tests/test_logger.py) for some ideas.